packages/html/src/drawer/tests/drawer-item-states.tsx

```diff
@@ -9,17 +10,44 @@ const styles = `
 export default () =>(
     <>
         <style>{styles}</style>
-        <div id="test-area">
-            <DrawerNormal items={
-                <>
-                    {[ 'normal', ...DrawerItem.states ].map((state) => (
-                        <DrawerItem text={`Item state: ${state}`} {...{ [state]: true }} icon="star-outline" />
-                    ))}
-                    <DrawerItem text="Item state: selected + hover" selected hover icon="star-outline" />
-                    <DrawerItem text="Item state: selected + focus" selected focus icon="star-outline" />
-                </>
-            }>
-            </DrawerNormal>
+        <div id="test-area" className="k-d-grid k-gap-6">
+            <span>Drawer with icons</span>
```

Comment:
Maybe we could change this to: Drawer with outline icons ?

Reply:
Does it matter? The idea here is that in the second case the theme color of the icons is changed, while here they are normal- it doesn't really matter if its and outline or solid icon

Reply:
I think it matters, but I'll leave it for you to decide

Reply:
In the second example, I've replaced the outline icon with solid so that the change in the color is more noticeable. I'll replace all icons for consistency.